testdata/generated/appsv1.go

```diff
@@ -0,0 +1,124 @@
+
+/*
+Copyright 2022 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Code auto-generated. DO NOT EDIT.
+// +build !ignore_autogenerated
+
+package generated
+
+import (
+	"ctx"
+	appsapiv1 "github.com/kcp-dev/client-gen/testdata/pkg/apis/apps/v1"
+	appsv1 "k8s.io/client-gen/kubernetes/typed/apps/v1"
+
+	"github.com/kcp-dev/kcp-client-wrappers/kcp"
```

Comment:
I think this needs to point to https://github.com/ncdc/kcp-client-wrappers for now since the library doesn't exist in kcp-dev

Reply:
actually that should end up in https://github.com/kcp-dev/apimachinery, I'll go ahead and make that PR

Reply:
Please update now that the kcp-dev/apimachinery PR has merged